src/BinSkim.Rules/RuleIds.cs

```diff
@@ -69,5 +69,8 @@ internal static class RuleIds
         // MachO Checks
         public const string EnablePositionIndependentExecutableMachO = "BA5001";
         public const string DoNotAllowExecutableStack = "BA5002";
+
+        // SourceLink Checks
+        public const string EnableSourceLink = "BA6001";
```

Comment:
>EnableSourceLink [](http://example.com/codeflow?start=28&length=16)

Let's go ahead and proceed with BA2027 and assume the security connection is sufficiently obvious to be in our core charter. :) As I believe it is. Efficient security response is important, it is a truism in our space that attackers have all the time in the world and, when something finally pops, every cycle counts for response engineers.

The primary/happy path value here, of course, is simply making API/dependency consumers happy, i.e., this change maximizes eco-system satisfaction/growth.